recipes/default.rb

```diff
@@ -18,4 +18,4 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-include_recipe 'newrelic-ng::nrsysmond-default'
+include_recipe cookbook_name + '::nrsysmond-default'
```

Comment:
Do you think it's in general a wise idea to never hardcode the cookbooks name anywhere?
Can you explain why this is advantageous - I've not seen this practise yet in other cookbooks.